pkgs/development/libraries/libcamera/default.nix

```diff
@@ -0,0 +1,85 @@
+{ stdenv
+, fetchgit
+, lib
+, meson
+, ninja
+, pkg-config
+, boost
+, gnutls
+, openssl
+, libevent
+, lttng-ust
+, libtiff
+, qt5
+, gst_all_1
+, graphviz
+, doxygen
+, python3
+, python3Packages
+}:
+
+stdenv.mkDerivation {
+  pname = "libcamera";
+  version = "1.0.0";
```

Comment:
In build files it listed as 0.0.0:

https://git.linuxtv.org/libcamera.git/tree/meson.build

So we should probably use the unstable version scheme.

Reply:
Done